### PR TITLE
Fixes corrupted state, when window is missing

### LINKED ISSUE #13134: When profile is corrupt, broken entries need to be pinned twice to stay pinned
When testing https://github.com/brave/browser-laptop/issues/13102, @kjozwiak and I noticed that the fix didn't solve the issue 100%.  

We had to do the following steps to "fix" the issue:

1. Have my profile (I have a backup- contact me; needs `session-store-1` and `ledger-state.json`)
2. Pin wikipedia.org
3. Visit wikipedia.org for ~10 seconds
4. Go to preferences#payments
5. Notice wikipedia.org got unpinned
6. Pin wikipedia.org again
7. Visit wikipedia.org for ~10 seconds
8. Go to preferences#payments
9. Notice that pin stayed this time 😄

### PR DESCRIPTION
**Pull request by NejcZdovc**
PR that fixes https://github.com/brave/browser-laptop/issues/13134

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -195,6 +195,11 @@ const Synopsis = function (options) {
         delete entry.score
       }
     }
+
+    if (entry.window == null) {
```
Comment (mrose17) [inline index.js]: to be consistent, can we do this on one-line, i.e.,

              if (!entry.window) entry.window = []

thanks!

**Review by mrose17 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (NejcZdovc) [reply index.js]: done

Comment (bsclifton) [reply index.js]: if you wanted to get fancy:
`entry.window = entry.window || []` 😛

**Review by mrose17 — APPROVED** [review]

**Comment by mrose17** [comment]
yes, @bsclifton - that's probably a better idiom... if only i asked to remove the blank line below it too!

**Pull request merged**